web-ui/src/main/java/org/archcnl/webui/mappings/Triplet.java

```diff
@@ -0,0 +1,80 @@
+package org.archcnl.webui.mappings;
+
+import org.archcnl.webui.exceptions.ObjectIsNotAConceptException;
+import org.archcnl.webui.exceptions.ObjectIsNotAVariableException;
+import org.archcnl.webui.exceptions.RecursiveRelationException;
+import org.archcnl.webui.exceptions.RelationCannotRelateToObjectException;
+
+public class Triplet {
```

Comment:
We need to Implement a Triplet Factory to remove the setter from the constructor.